app/frontend/src/components/forms/manage/TeamManagement.vue

```diff
@@ -57,15 +57,39 @@
       no-data-text="Failed to load team role data"
       :search="search"
     >
+      <!-- custom header markup - add tooltip to roles -->
+      <template v-for="h in headers.slice(2)" v-slot:[`header.${h.value}`]="{ headers }">
```

Comment:
i know this can be done a better way.. but i'm struggling to get my head around slot properties

Reply:
Given how this table is designed to be extensible, I think the way this is implemented is acceptable.